docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst

```diff
@@ -21,15 +21,15 @@ and vCenter infrastructure. You can install pyVmomi using pip:
 
 .. code-block:: bash
 
-    $ pip install pyvmomi
```

Comment:
Add a not that you can use pip as well.

Reply:
I'm curious why this is necessary (changing pip to pip3) - doesn't it depend on the distro? For example, I'm running Fedora 32 and it isn't called pip3... it's still just pip (though running python 3.8).